src/GameLogic.cpp

```diff
@@ -60,23 +87,23 @@ Player& GameLogic::getPlayer() {
     return player;
 }
 
-std::vector<Actor>& GameLogic::getActors() {
+std::vector<std::reference_wrapper<Actor>> GameLogic::getActors() {
```

Comment:
Do we still need to return the actual reference to the actors vector, or does just a copy suffice?

Reply:
I think a reference to it will use less memory overhead, I am not sure though.